apps/emqx_bridge_azure_event_hub/src/emqx_bridge_azure_event_hub.erl

```diff
@@ -166,14 +166,14 @@ values(producer) ->
 %% `emqx_bridge_resource' API
 %%-------------------------------------------------------------------------------------------------
 
-connector_config(Config, BridgeName) ->
+connector_config(Config, _BridgeName) ->
```

Comment:
Now that the bridge name is always inside `Config`, perhaps we could change the callback API to `connector_config/1`.

Reply:
Fixed here: https://github.com/emqx/emqx/pull/11551